ext/standard/tests/array/array_unshift_variation5.phpt

```diff
@@ -46,13 +46,9 @@ $arrays = array (
        array("1" => 1),
        array("one" => 1, 'two' => 2, "three" => 3, 4 => 4),
 
-       // arrays with float values
-/*5*/  array("float" => 2.3333),
-       array("f1" => 1.2, 'f2' => 3.33, 3 => 4.89999922839999, 'f4' => 33333333.333333),
```

Comment:
Not particularly important, but these two look unrelated? Those are float values only.

Reply:
Indeed, don't know why I went and removed those